scripts/projectView.js

```diff
@@ -1,87 +1,90 @@
-    if ($('#title-filter option[value="' + val + '"]').length === 0) {
-      $('#title-filter').append(optionTag);
-    }
+  projectView.populateFilters = function() {
+    $('article').each(function() {
+      // if(!$(this).hasClass('.template')){
+      val = $(this).attr('data-title');
+      optionTag = '<option value="' + val + '">' + val + '</option>';
+      if ($('#title-filter option[value="' + val + '"]').length === 0) {
+        $('#title-filter').append(optionTag);
+      }
 
-    val = $(this).attr('data-category');
-    optionTag = '<option value="' + val + '">' + val + '</option>';
-    if ($('#category-filter option[value="' + val + '"]').length === 0) {
-      $('#category-filter').append(optionTag);
-    }
-    // }
-  });
-};
+      val = $(this).attr('data-category');
+      optionTag = '<option value="' + val + '">' + val + '</option>';
+      if ($('#category-filter option[value="' + val + '"]').length === 0) {
+        $('#category-filter').append(optionTag);
+      }
+      // }
+    });
+  };
 
-projectView.handleTitleFilter = function() {
-  $('#title-filter').on('change', function () {
-    // console.log('selected Val ' + $(this).val());
-    if($(this).val()) {
-      var $articles = $('article');
-      var $selectVal = $(this).val();
-      $articles.hide();
-      $articles.each(function(){
-        if($(this).attr('data-title') === $selectVal) {
+  projectView.handleTitleFilter = function() {
+    $('#title-filter').on('change', function () {
+      // console.log('selected Val ' + $(this).val());
+      if($(this).val()) {
+        var $articles = $('article');
+        var $selectVal = $(this).val();
+        $articles.hide();
+        $articles.each(function(){
+          if($(this).attr('data-title') === $selectVal) {
+            $(this).fadeIn();
+          }
+        });
+      } else {
+        $('article').each(function() {
           $(this).fadeIn();
-        }
-      });
-    } else {
-      $('article').each(function() {
-        $(this).fadeIn();
-      });
-    };
-    $('#category-filter').val('');
-  });
-};
+        });
+      };
+      $('#category-filter').val('');
+    });
+  };
 
-projectView.handleCategoryFilter = function() {
-  $('#category-filter').on('change', function () {
-    if($(this).val()) {
-      var $articles = $('article');
-      var $selectVal = $(this).val();
-      $articles.hide();
-      $articles.each(function(){
-        if($(this).attr('data-category') === $selectVal) {
+  projectView.handleCategoryFilter = function() {
+    $('#category-filter').on('change', function () {
+      if($(this).val()) {
+        var $articles = $('article');
+        var $selectVal = $(this).val();
+        $articles.hide();
+        $articles.each(function(){
+          if($(this).attr('data-category') === $selectVal) {
+            $(this).fadeIn();
+          }
+        });
+      } else {
+        $('article').each(function() {
           $(this).fadeIn();
```

Comment:
Not sure if fadeIn() will do that for me. I think i need to keep that .each() so i can itetate thru all articles but I will look into that.